Neos.Neos/Tests/Behavior/Features/FrontendRouting/DisableNodes.feature

```diff
@@ -158,37 +158,6 @@ Feature: Routing behavior of removed, disabled and re-enabled nodes
     And The node "sir-david-nodenborough" in content stream "cs-identifier" and dimension "{}" should resolve to URL "/david-nodenborough"
     And The node "earl-o-documentbourgh" in content stream "cs-identifier" and dimension "{}" should resolve to URL "/david-nodenborough/earl-document"
 
-  Scenario: Disable the same node twice
-    When the command "DisableNodeAggregate" is executed with payload:
-      | Key                          | Value                    |
-      | contentStreamId              | "cs-identifier"          |
-      | nodeAggregateId              | "sir-david-nodenborough" |
-      | coveredDimensionSpacePoint   | {}                       |
-      | nodeVariantSelectionStrategy | "allVariants"            |
-    And the event NodeAggregateWasDisabled was published with payload:
-      | Key                          | Value                    |
-      | contentStreamId              | "cs-identifier"          |
-      | nodeAggregateId              | "sir-david-nodenborough" |
-      | affectedDimensionSpacePoints | [{}]                     |
```

Comment:
so this state should be impossible, is thats why this test was removed?

Reply:
yep, this is to be caught by constraint checks